### ISSUE TITLE
Devicons: code point ("Gopher") overwritten by Greek symbol ("Sigma")

### ISSUE THREAD

**Issue by baumanno** [gabrielelana/awesome-terminal-fonts#52]
Hi,
I'm not quite sure if this is a borked setup on my side or something wrong with this package/one of its dependencies, but my hope is we can figure this out here together rather than me rage-typing into search engines.

I'm running Arch Linux and currently using the AUR package provided, which builds straight from this repo; however, the official package shows the same behaviour.

Recently (~1 week ago) I noticed two icons I was using in my statusbar differed from what they should be, namely the lower-case lambda, and the Golang Gopher icon provided by the Devicons icon set. This probably co-incided with one of the semi-regular VCS rebuilds I run on my system, which rebuilds all packages based directly off an upstream repo, including this one.

The lambda came back (somehow, don't know how, don't really mind), yet the Gopher is still missing. According to [the cheatsheet](https://vorillaz.github.io/devicons/#/cheat), the icon uses codepoint \e624. When I type `echo "\ue624"`, and when I look at my statusbar, this shows the greek upper-case Sigma:

![2018-03-02-124308_197x63_scrot](https://user-images.githubusercontent.com/1183339/36897846-8aac3942-1e18-11e8-96ac-da1aa0210dc1.png)

 Incidentally, I see similar behaviour for all other code points in Devicons, Octicons and Pomicons, yet FontAwesome works just fine.

Any idea what this could be related to? I'm CC'ing @shibumi as they provided the Arch package and might know the ins and outs.
My hunch is that I have some font that is using those code points, but I did not consciously install any...
Thanks, and have a great day! :)

**Comment by shibumi** [comment]
Fonts on Linux are pretty awful :)
I have to admit I have no idea what's going on. I guess your powerline zsh plugin uses other codepoints for their icons. Which one do you use?

**Comment by baumanno** [comment]
Yeah, I tend to agree ;-)
I doubt this has anything to do with zsh, I use an iconic font for the i3status workspace-icons. So this issue is present both in the terminal and in i3status, which leads me to believe it's system-wide. In my i3-config I have:
```
...
set $ws4 "4:λ" # lambda as stated above
set $ws5 "5"    # should be Gopher, is blank, was Sigma
set $ws6 "6:" # terminal-icon
...
bar {
    status_command i3status -c /home/baumanno/.config/i3status/config
    font pango:Inconsolata, FontAwesome 7
}
```
Hmm... is there a way to find out what fonts are currently providing glyphs for the conflicting codepoints? Then I could boil it down to some font and just see if removing that fixes the issue. Thanks for the response btw! :)

**Comment by baumanno** [comment]
**TL;DR** the fonts provided with the [mathjax-package](https://www.archlinux.org/packages/community/any/mathjax/) were conflicting with devicons. Removing mathjax resolved the issue.

Figuring this out turned out to be simpler than I thought. Using [this script](https://unix.stackexchange.com/a/268286), I figured out what fonts on my system were claiming the conflicting codepoint: `./script.py U+e624`

This listed the obvious `devicons.ttf` and the not-quite-so-obvious `LatinModernMathJax_NonUnicode-Regular.woff`. This font is distributed with the Arch mathjax-package, which in turn is a dependency of jupyter-notebooks.

I removed mathjax and jupyter and shall henceforth run jupyter exclusively in virtualenvs.

Which is what I've been doing anyway :eyes:
